src/Core/MailTemplate/MailLayoutFolderCatalog.php

```diff
@@ -26,7 +26,7 @@
 
 namespace PrestaShop\PrestaShop\Core\MailTemplate;
 
-use PrestaShop\PrestaShop\Core\Exception\InvalidException;
+use PrestaShop\PrestaShop\Core\Exception\FileNotFoundException;
```

Comment:
An exception already exist in Symfony:
`Symfony\Component\Filesystem\Exception\FileNotFoundException` do not reinvent the wheel :)

Reply:
I know but this exception is scoped to the symfony filesystem component 
And I want our Exception to inherit ou `CoreException` so that we can easily catch a PrestaShop exception

Reply:
Oohhh, I understand! Thanks :smiley: